### PR TITLE
Add description to the HandInteractionRecordArticulatedHandPose scene

### LINKED ISSUE #9385: Hand Interaction Record Articulated Hand Pose example scene does not have documentation
## Describe the bug

Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionRecordArticulatedHandPose.unity does not have a scene description panel, or any documentation that tells users about what the scene does.

The Recorder has voice commands for recording the Articulate Hands input, but does not have an obvious use for the sphere and cube that are in the scene.

### PR DESCRIPTION
**Pull request by MaxWang-MS**
Fixes #9385

### CODE REVIEW

File: Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionRecordArticulatedHandPose.unity [hunk 1]
```diff
@@ -437,6 +431,353 @@ Camera:
...
+      propertyPath: m_textInfo.pageCount
+      value: 1
+      objectReference: {fileID: 0}
+    - target: {fileID: 2113875699234556495, guid: 18470d71939382448be2ecd06efa9662,
+        type: 3}
+      propertyPath: m_text
+      value: '<size=32><b>Record Articulated Hand Pose Example</b></size>
+
+
+        This example scene demonstrates a way to record articulated hand poses usiung
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionRecordArticulatedHandPose.unity:522]:
```suggestion
        This example scene demonstrates a way to record articulated hand poses using
```

File: Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionRecordArticulatedHandPose.unity [hunk 2]
```diff
@@ -437,6 +431,353 @@ Camera:
...
+        MRTK. To do that, say either "record left hand" or "record right hand" and
+        then you will see the color of the sphere or cube changes respectively, bringing
+        you to the ready-to-capture state. To record/capture the current hand pose,
+        say "stop" and the current hand pose will be saved to the HoloLens 2 device.
+
+
+        Saved hand pose can be accessed through the device portal at User Folders/LocalAppData/[APP_NAME]/LocalState/
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionRecordArticulatedHandPose.unity:529]:
```suggestion
        Saved hand poses can be accessed through the device portal at User Folders/LocalAppData/[APP_NAME]/LocalState/
```

File: Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionRecordArticulatedHandPose.unity [hunk 3]
```diff
@@ -620,8 +961,8 @@ Transform:
   m_PrefabAsset: {fileID: 0}
   m_GameObject: {fileID: 988162582}
   m_LocalRotation: {x: 0, y: 0, z: 0, w: 1}
-  m_LocalPosition: {x: -0.84, y: -0.26137352, z: 4.36}
-  m_LocalScale: {x: 0.4, y: 0.4, z: 0.4}
+  m_LocalPosition: {x: 0.158, y: -0.26137352, z: 1.58}
+  m_LocalScale: {x: 0.2, y: 0.2, z: 0.2}
```
Comment (keveleigh) [inline Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionRecordArticulatedHandPose.unity:965]: Are these changes expected?

**Review by keveleigh — APPROVED** [review]

File: Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionRecordArticulatedHandPose.unity (hunk 3 above)
Comment (MaxWang-MS) [reply Assets/MRTK/Examples/Demos/HandTracking/Scenes/HandInteractionRecordArticulatedHandPose.unity:965]: Yes they are intentional.

**Pull request merged**